i6_models/primitives/feature_extraction.py

```diff
@@ -22,6 +22,9 @@ class LogMelFeatureExtractionV1Config(ModelConfiguration):
         min_amp: minimum amplitude for safe log
         num_filters: number of mel windows
         center: centered STFT with automatic padding
+        periodic: whether the window is assumed to be periodic
+        mel_options: extra options for mel filters
+        rasr_compatible: apply FFT to make features compatible to RASR's
```

Comment:
You should add sth like:
```suggestion
        rasr_compatible: apply FFT to make features compatible to RASR's,
           otherwise (default) apply STFT
```